Sprint-1/4-stretch-explore/chrome.md

```diff
@@ -11,8 +11,13 @@ In the Chrome console,
 invoke the function `alert` with an input string of `"Hello world!"`;
 
 What effect does calling the `alert` function have?
+    it will trigger an alert box that says "Hello world!" on the page.
 
 Now try invoking the function `prompt` with a string input of `"What is your name?"` - store the return value of your call to `prompt` in an variable called `myName`.
 
 What effect does calling the `prompt` function have?
+    it will trigger a dialog box with a message and an input field 
+
 What is the return value of `prompt`?
+    if we entered a value on the input field and pressed ok the value we enterned will retun 
```

Comment:
Good stuff! I think if you try it out, pressing OK without entering a value into the prompt will assign an empty string (`""`) to the variable `myName`, whereas pressing Cancel will as you say assign `null`. But this is something you'll probably check when you need to know, not something that you need to remember! (In practice, people will use `prompt` very rarely!)